src/Pages/Login/Login.js

```diff
@@ -1,83 +1,125 @@
 import React, { useState } from 'react';
-import { loginUser } from '../../APIFunctions/Auth';
-import Background from '../../Components/Background/background';
 import { useAuth } from '../../Components/context/AuthContext';
+import { loginUser } from '../../APIFunctions/Auth';
 
-export default function Login() {
+const LoginForm = ({ className, ...props }) => {
```

Comment:
just put the ui in the Login component,
vibe coding is best with understanding of the original code, and the minimum possible changes in the pr